src/utils/webhook.ts

```diff
@@ -69,67 +62,11 @@ export const sendWebhookRequest = async (
       status: resp.status,
       body: await resp.text(),
     };
-  } catch (e: any) {
+  } catch (e) {
     return {
       ok: false,
       status: 500,
-      body: e.toString(),
+      body: prettifyError(e),
     };
   }
 };
-
-// TODO: Add retry logic upto
```

Comment:
This was legacy from v1 and unused. Migrated to v2 queues and trigger it from `mineTransactionWorker`